client/rpc-spec-v2/src/chain_head/chain_head.rs

```diff
@@ -250,33 +250,71 @@ where
 	}
 }
 
+/// Internal representation of the initial blocks that should be
+/// reported or ignored by the chainHead.
+#[derive(Clone, Debug)]
+struct InitialBlocks<Block: BlockT> {
+	/// Children of the latest finalized block, for which the `NewBlock`
+	/// event must be generated.
+	///
+	/// It is a tuple of (block hash, parent hash).
+	in_memory: Vec<(Block::Hash, Block::Hash)>,
```

Comment:
nit: I think `in_memory` is not very descriptive here. Maybe `finalized_block_descendants`?